RetrospectiveExtension.Frontend/utilities/external/telemetryclient.ts

```diff
@@ -52,10 +52,10 @@ export class TelemetryClient {
     this.ExtensionContext = settings.extensioncontext;
 
     try {
-      const webContext = VSS.getWebContext();
+      const user = SDK.getUser();
 
       AppInsights.downloadAndSetup(config);
-      AppInsights.setAuthenticatedUserContext(webContext.user.id, webContext.collection.id);
+      AppInsights.setAuthenticatedUserContext(user.id, user.name);
```

Comment:
what is the difference between `collection.id` and `user.name`? is this an intended change?

Reply:
Yes, they are. One is the unique id and the other is email address.